cmake/linux/package_linux.sh.in

```diff
@@ -0,0 +1,161 @@
+
+# Clone install to AppDir
+echo -e "\nCopying @CMAKE_INSTALL_PREFIX@ to ${APPDIR}..."
+cp -R "@CMAKE_INSTALL_PREFIX@/." "${APPDIR}usr"
+rm -rf "${APPDIR}usr/include"
+success "${APPDIR}"
+
+# Create a wrapper script which calls the lmms executable
+mv "${APPDIR}usr/bin/lmms" "${APPDIR}usr/bin/lmms.real"
+# shellcheck disable=SC1083
+cat >"${APPDIR}usr/bin/lmms" <<EOL
+#!/usr/bin/env bash
+DIR="\$( cd "\$( dirname "\${BASH_SOURCE[0]}" )" && pwd )"
+export LD_LIBRARY_PATH=\$DIR/usr/lib/:\$DIR/usr/lib/lmms:\$LD_LIBRARY_PATH
+# Prevent segfault on VirualBox
+if lsmod |grep vboxguest > /dev/null 2>&1; then
+   echo "VirtualBox detected.  Forcing libgl software rendering."
+   export LIBGL_ALWAYS_SOFTWARE=1;
+fi
+QT_X11_NO_NATIVE_MENUBAR=1 \$DIR/usr/bin/lmms.real "\$@"
+EOL
+
+chmod +x "${APPDIR}usr/bin/lmms"
+
+# Per https://github.com/probonopd/linuxdeployqt/issues/129 
+unset LD_LIBRARY_PATH
+
+# Ensure linuxdeployqt can find shared objects
+export LD_LIBRARY_PATH="${APPDIR}usr/lib/lmms/":$LD_LIBRARY_PATH
+
+# Handle wine linking
+if [ -d "@WINE_LIBRARY_FIX@" ]; then
+   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:@WINE_LIBRARY_FIX@:@WINE_LIBRARY_FIX@wine/
+fi
+
+# Move executables so linuxdeployqt can find them
+ZYNLIB="${APPDIR}usr/lib/lmms/RemoteZynAddSubFx"
+VSTLIB="${APPDIR}usr/lib/lmms/RemoteVstPlugin.exe.so"
+
+ZYNBIN="${APPDIR}usr/bin/RemoteZynAddSubFx"
+VSTBIN="${APPDIR}usr/bin/RemoteVstPlugin.exe.so"
+
+mv "$ZYNLIB" "$ZYNBIN"
+mv "$VSTLIB" "$VSTBIN"
+
+# Patch the desktop file
+sed -i 's/.*Exec=.*/Exec=lmms.real/' "$DESKTOPFILE"
+
+# Build a list of shared objects in target/lib/lmms
+for file in "${APPDIR}usr/lib/lmms/"*.so; do
+   thisfile="${APPDIR}usr/lib/lmms/${file##*/}"
+   executables="${executables} -executable=$thisfile"
+done
+
+# Fix ladspa linking
+executables="${executables} -executable=${APPDIR}/usr/lib/lmms/ladspa/imp_1199.so"
+executables="${executables} -executable=${APPDIR}/usr/lib/lmms/ladspa/imbeq_1197.so"
+executables="${executables} -executable=${APPDIR}/usr/lib/lmms/ladspa/pitch_scale_1193.so"
+executables="${executables} -executable=${APPDIR}/usr/lib/lmms/ladspa/pitch_scale_1194.so"
+
+# Bundle both qt and non-qt dependencies into appimage format
+echo -e "\nBundling and relinking system dependencies..."
+echo -e "#### linuxdeployqt pass 1 ####" > "$LOGFILE"
+# shellcheck disable=SC2086
+"$LINUXDEPLOYQT" "$DESKTOPFILE" -executable="$ZYNBIN" -executable="$VSTBIN" $executables -bundle-non-qt-libs -verbose=$VERBOSITY $STRIP >> "$LOGFILE" 2>&1
+success "Bundled and relinked dependencies"
+
+# Link to original location so lmms can find them
+ln -sr  "$ZYNBIN" "$ZYNLIB"
+ln -sr  "$VSTBIN" "$VSTLIB"
+
+# Run a second time to create AppImage
+echo -e "\nFinishing the AppImage..."
+echo -e "\n\n#### linuxdeployqt pass 2 ####" >> "$LOGFILE"
+# shellcheck disable=SC2086
+"$LINUXDEPLOYQT" "$DESKTOPFILE" -executable="$ZYNBIN" -executable="$VSTBIN" $executables -appimage -verbose=$VERBOSITY $STRIP >> "$LOGFILE" 2>&1
```

Comment:
This concerns me, because the second call to `linuxdeployqt` is done for "known issues" during image creation.  Are you suggesting we call `linuxdeployqt` twice, then `appimagetool`?